internal/scms/azuredevops/token_manager_test.go

```diff
@@ -0,0 +1,146 @@
+package azuredevops
```

Comment:
Can we get these tests to used Ginkgo an example can be seen there that uses it as a simple test runner without setting up controllers etc: https://github.com/argoproj-labs/gitops-promoter/blob/main/internal/utils/utils_test.go